src/js/containers/search/visualizations/rank/SpendingByAwardingAgencyVisualizationContainer.jsx

```diff
@@ -184,7 +185,8 @@ export class SpendingByAwardingAgencyVisualizationContainer extends React.Compon
         // generate the API parameters
         const apiParams = {
             field: 'transaction_obligated_amount',
-            group: `award__awarding_agency__${this.state.agencyScope}_agency__name`,
+            group: [`award__awarding_agency__${this.state.agencyScope}_agency__name`,
+                `treasury_account__awarding_toptier_agency__abbreviation`],
```

Comment:
On this endpoint, `treasury_account__awarding_subtier_agency` does not exist.  I will remove the field from the query if it is subtier.